traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -56,6 +64,8 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		{1.2, http.MethodGet, "regions/{id}$", regionsHandler(d.DB), RegionsPrivLevel, Authenticated, nil},
 		//Servers
 		{1.2, http.MethodGet, `servers(\.json)?$`, serversHandler(d.DB), ServersPrivLevel, Authenticated, nil},
+		// explicitly passed to legacy system until fully implemented
+		{1.2, http.MethodGet, "servers/status$", proxyHandlerFunc, ServersPrivLevel, Authenticated, []Middleware{}},
```

Comment:
Since perl TO will be handling authentication and authorization I think this endpoint should `use 0, NoAuth` instead of `ServersPrivLevel, Authenticated`

Reply:
excellent point..